### ISSUE TITLE
Web [Charts]: Cannot navigate bar charts with screen reader

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#40873]
I am unable to navigate into the chart to explore the groups and values with a screen reader. When drilling into the chart, the screen reader only announces "No visible title to interact" with no further elements that are explorable.

This happens on both dashboard insights and search aggregation insights.

https://user-images.githubusercontent.com/10532611/186774016-1c4bbf7d-7eff-4b81-a7c6-f4d1f1064c6e.mov

/cc @joelkw @felixfbecker @vovakulikov @unclejustin
